.github/labels.yml

```diff
@@ -229,6 +235,15 @@
 - name: status/done
   color: edb3a6
   description: "Status: Done"
+- name: status/Approved-Waiting
```

Comment:
Please follow capitalisation for label names: small case, with hyphens for spaces: `status/approved-waiting`.